### PR TITLE
replaced cobra with kingpin for the tsdb cli tool

### PR DESCRIPTION
**Pull request by krasi-georgiev**
#107 as part of extending the cli tool replacing cobra with kingpin to be consistent with the other packages
Signed-off-by: Krasi Georgiev <krasi.root@gmail.com>

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
cc @Gouthamve

File: cmd/tsdb/main.go [hunk 1]
```diff
@@ -33,35 +31,30 @@ import (
...
-	root := &cobra.Command{
-		Use:   "tsdb",
-		Short: "CLI tool for tsdb",
-	}
-
-	root.AddCommand(
-		NewBenchCommand(),
-	)
+var (
+	cli = kingpin.New(filepath.Base(os.Args[0]), "CLI tool for tsdb")
```
Comment (Gouthamve) [inline cmd/tsdb/main.go]: Can you move this into main() itself? You could also make benchWriteFile a member of the `writeBenchmark` struct and that way you don't need the global variable.

**Comment by krasi-georgiev** [comment]
@Gouthamve  no problem , have a look now. 
I left in `var()` block as `gofmt` formats the assignment easier to read this way with tabular indentation.

**Comment by Gouthamve** [comment]
LGTM!

**Pull request merged**